src/components/app/DetailsContainer.js

```diff
@@ -47,13 +44,15 @@ function DetailsContainer({
       secondaryInitialSize={20}
       onDragEnd={invalidatePanelLayout}
     >
-      <Details />
-      {Sidebar && <Sidebar />}
+      <>
+        <Details />
+        {Sidebar && <Sidebar />}
+      </>
```

Comment:
You'll want to check that this still works as expected; I remember that SplitterLayout especially works with the direct children.

Reply:
Yeah, this was giving me an error. It didn't like using `false` as a child. I changed it to a ternary that applies the component or null.